pyramid_oereb/contrib/data_sources/oereblex/sources/document.py

```diff
@@ -199,12 +199,16 @@ def _get_document_records(self, document, language):
             raise AssertionError('Missing authority for document #{0}'.format(document.id))
         if oereblex_doctype is None:
             raise AssertionError('Missing type for document #{0}'.format(document.id))
+        if document.authority_url is None:
```

Comment:
Isn't this a mandatory attribute?

Reply:
At least not as it is defined now: https://github.com/openoereb/pyramid_oereb/blob/master/pyramid_oereb/core/records/office.py#L21-L36

Reply:
Ah I see. There is a missmatch between Document Record and Office Record.